src/Header/Header.stories.tsx

```diff
@@ -64,6 +65,37 @@ export const DesktopDark: Story = {
   },
 };
 
+export const WithSubNav: Story = {
+  render: (args) => (
+    <>
+      <Header {...args} />
+      <SubNav
+        variant="dark"
+        label="features"
+        items={[
+          { id: '1', label: 'UI Tests', href: '/ui-tests' },
+          {
+            id: '2',
+            label: 'Visual test',
+            href: '/visual-tests',
+            isActive: true,
+          },
+          { id: '3', label: 'Interaction test', href: '/interaction-tests' },
+          { id: '4', label: 'TurboSnap', href: '/turbosnap', external: true },
```

Comment:
Is the `id` necessary? Can't you use the array index directly?

Reply:
usually not a good idea to use index. There is even an eslint rule that prevents it.

Reply:
Ah yes you're right.